.github/workflows/windows-msvc.yml

```diff
@@ -205,6 +205,7 @@ jobs:
           sed -i '/AT_SETUP(\[BCD ADD and SUBTRACT, DEFAULT ROUNDING MODE\])/a AT_SKIP_IF(\[true\])' testsuite.src/data_packed.at
           sed -i '/AT_SETUP(\[BCD ADD and SUBTRACT, all ROUNDED MODEs\])/a AT_SKIP_IF(\[true\])' testsuite.src/data_packed.at
           sed -i '/AT_SETUP(\[CURRENCY SIGN WITH PICTURE SYMBOL\])/a AT_SKIP_IF(\[true\])' testsuite.src/run_misc.at
+          sed -i '/AT_SETUP(\[PROGRAM COLLATING SEQUENCE\])/a AT_SKIP_IF(\[true\])' testsuite.src/syn_definition.at
```

Comment:
@GitMensch This new exception is needed because of commit [5310](https://sourceforge.net/p/gnucobol/code/5310/). To avoid these "surprises", would it be reasonable to check SVN-targetted modifications with this GitHub CI before committing to SVN ?

Reply:
To avoid these surprises we should do the right thing - drop all those skips and instead tell Windows to not do the popup but error until told otherwise via a special setting.

We discussed that before and have checked the msdn for the necessary function call - so far no one has implemented that.